.github/workflows/integration-tests.yaml

```diff
@@ -21,7 +21,6 @@ jobs:
           - "2022.09.0" # bionic
           - "2022.03.2"
           - "2021.09.0"
-          - "1.8.8.2"
```

Comment:
Why drop this one?

Reply:
Some of the old integration tests, when applied to the new thumbnail methods, work on all versions listed except 1.8.8.2.

Usually removing a version because tests fail is definitely a no-go, but this is way outside of our support window — I checked [here](https://posit.co/support/#supported-pro-versions) and the oldest supported version is 2023.03. I assumed that its inclusion was an oversight and removed it.

Reply:
Can we just skip those tests on older Connect versions? We have a test helper for that IIRC. 

I know this is outside of our support window, but if it's not a burden to us (parallel CI is amazing), I'd like to keep as broad support as is reasonable. We do have many customers running Connect beyond the support window, and our SDKs can be a useful tool to help them manage and upgrade their content to help with upgrading Connect itself.

Reply:
Sure thing! Keeping support as broad as possible makes sense, especially given that the SDKs are helpful when people need to migrate to newer, supported versions. I'll look into skipping those tests on 1.8.8.2.

Reply:
The failing test is skipped now (technically two tests, but both wind up calling `set_thumbnail()` with a remote path to Connect's favicon). Much better than omitting _all_ tests on that version. :)